src/Server/Services/Analytics/AnalyticsService.lua

```diff
@@ -768,13 +1038,13 @@ end
 		score = 400 -- Completed the mission with a score of 400
 	})
 	```
-	
-	For more information on progression events, refer to [GameAnalytics docs](https://docs.gameanalytics.com/integrations/sdk/roblox/event-tracking?_highlight=teleportdata#progression) on progression.
-	
+
+	For more information about progression events, refer to [GameAnalytics docs](https://docs.gameanalytics.com/integrations/sdk/roblox/event-tracking?_highlight=teleportdata#progression) on progression.
+
 	@param data ProgressionEvent
-	@return { [any]: any }
+	@return Promise
```

Comment:
```suggestion
	@return Promise<T>
```
Unless the type is already defined, you need to define custom return types with Moonwave by denoting them with a `<T>` after the type (Eg. `Promise<T>`)